src/main/java/InterfaceAdapters/ScheduleManager.java

```diff
@@ -18,56 +19,63 @@
 
 public class ScheduleManager {
 
-    // attempts to fix these style warnings have resulted in problems in other classes
-    private HashMap<Person, Schedule> schedules;
-    private final ScheduleEditor editor;
-    private final ScheduleComparer comparer;
+//    // attempts to fix these style warnings have resulted in problems in other classes
+//    private HashMap<Person, Schedule> schedules;
+    private ScheduleEditor editor;
+    private ScheduleComparer comparer;
+    private UserList data;
+    private FriendAdder adder;
 
     public ScheduleManager() {
         editor = new ScheduleEditor();
         comparer = new ScheduleComparer();
-        schedules = new HashMap<>();
+//        schedules = new HashMap<>();
     }
 
-    /**
-     * Adds an instance of Person and Schedule to the schedules hashmap
-     * @param user1 the main user's Person
-     * @param user2 the friend's Person
-     */
+//    /**
+//     * Compares two people's schedules and returns a schedule showing what times both
+//     * are busy.
+//     * @param user1 the main user's Person
+//     * @param user2 the friend's Person
+//     */
+    // currently uncertain how we're going to be using this class
```

Comment:
wait are you suggesting we delete this from ScheduleManager or just change the implementation here?

Reply:
I believe that @tatDennis said that we might not need schedule comparer at all, so i'm commenting it out for now until we figure out if we do need it. if we don't need schedule comparer, then we don't need schedule manager to have a compare method